NOTE_FOR_DEVELOPERS.txt

```diff
@@ -0,0 +1 @@
+1. Every time a glade is edited with Glade editor for GTK3 (versions 3.9, 3.10, 3.11, etc.) the fixup scripts from dev-tools need to be run for the modified glade file. However if Glade 3.8 is used the fixup scripts are no longer required.
```

Comment:
maybe add something to Travis-CI run that tries to do the fixup, and if someone forgets, fail the build and cat this text-file/message to make sure things don't slip through

Reply:
@commodo I've added a pre build script that does just that. Let me know if it's ok with you.